### DISCUSSION TITLE
CTk.Image missing in the Documentation

### DISCUSSION THREAD

**Discussion by Programmieren-mit-Kopf** [TomSchimansky/CustomTkinter#1572] [Q&A] [answered]
Hey all,

I really like working with the documentation website https://customtkinter.tomschimansky.com/.
But unfortunatly I noticed, that the description of the CTk.Image is missing.

Is there a reason for that?

thanks in advance! :)

Kind Regards
Programmieren mit Kopf

**Comment by TomSchimansky** [comment]
Youre right, I will add CTkImage and CTkFont.

**Reply by avalon60** [reply]
That one caught me out also. It seemed to disappear after the docs got migrated.

**Comment by TomSchimansky** [comment, marked as answer]
Its now in the documentation again.

**Reply by Programmieren-mit-Kopf** [reply]
cool! thanks a lot :)
